Shared/Samples/Create convex hull around points/CreateConvexHullAroundPointsView.swift

```diff
@@ -0,0 +1,125 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct CreateConvexHullAroundPointsView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    var body: some View {
+        // Create a map view to display the map.
+        MapView(map: model.map, graphicsOverlays: model.graphicsOverlays)
+            .onSingleTapGesture { _, mapPoint in
+                model.inputPoints.append(mapPoint)
+                model.pointsGraphicsOverlay.addGraphic(Graphic(geometry: mapPoint, symbol: model.markerSymbol))
+                model.createIsDisabled = false
+                model.resetIsDisabled = false
+            }
+            .toolbar {
+                // Create button for making the convex hull.
+                ToolbarItem(placement: .bottomBar) {
+                    Button("Create") {
+                        model.createConvexHull()
+                    }
+                    .disabled(model.createIsDisabled)
+                }
+                // Reset button for removing all the points on the map.
+                ToolbarItem(placement: .bottomBar) {
+                    Button("Reset") {
+                        model.inputPoints.removeAll()
+                        model.pointsGraphicsOverlay.removeAllGraphics()
+                        model.convexHullGraphicsOverlay.removeAllGraphics()
+                        model.createIsDisabled = true
+                        model.resetIsDisabled = true
+                    }
+                    .disabled(model.resetIsDisabled)
+                }
```

Comment:
Can be [`ToolbarItemGroup`](https://developer.apple.com/documentation/swiftui/toolbaritemgroup)

Edit: if the 1 button suggestion below is applied then no need.